### DISCUSSION TITLE
How can I redirect http to https?

### DISCUSSION THREAD

**Discussion by frrobert2** [datenstrom/community#543] [Ask a question] [answered]
I have recently updated my yellow based site from http to https.  I am on shared hosting using apache.

I updated my .htaccess file and anything outside of yellow will properly redirect.
For example http://www.mysite.com/outsideyellow.html will redirect to https://www.mysite.com/outsideyellow.html

Anything inside yellow such as http://www.mysite.com/blog gives  a 404 page rather than redirecting to  https://www.mysite.com/blog 

Where should I add the redirect statements within yellow to get the redirects to function and what format should I use?

Thank you

**Comment by markseuffert** [comment, marked as answer]
Have a look at datenstrom/community#272

**Comment by frrobert2** [comment]
Thanks,  Sorry I didn't find 272.  Some how I missed it it in my search.
